sklearn/linear_model/logistic.py

```diff
@@ -1525,9 +1525,6 @@ def fit(self, X, y, sample_weight=None):
         cv = check_cv(self.cv, y, classifier=True)
         folds = list(cv.split(X, y))
 
-        self._enc = LabelEncoder()
-        self._enc.fit(y)
```

Comment:
It was added here - https://github.com/scikit-learn/scikit-learn/commit/67585f6e6f1ee5912fb520befe241fa74db8c37d, but not used anywhere... I am unable to understand why though...